mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx

```diff
@@ -632,7 +632,9 @@ export function PortfolioScreen({ navigation }: Props): JSX.Element {
           isSorted={isSorted}
           denominationCurrency={denominationCurrency}
         />
-        <DFIBalanceCard denominationCurrency={denominationCurrency} />
+        {activeButtonGroup === "ALL_TOKENS" && (
```

Comment:
What is this condition for?

Reply:
So that the DFI card only shows in the All tokens filter instead of all the filters (i.e. LP tokens, Crypto and dToken)

Reply:
can use provided enum

```suggestion
        {activeButtonGroup === ButtonGroupTabKey,AllTokens && (
```